docs/.vuepress/public-path.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+;(function() {
+  var ipfsPathRegExp = /^(\/(?:ipfs|ipns)\/[^/]+)/
+  var ipfsPathPrefix =
+    (window.location.pathname.match(ipfsPathRegExp) || [])[1] || ''
+
+  __webpack_public_path__ = ipfsPathPrefix + '/'
+  if (typeof window !== 'undefined') {
+    window.__VUEPRESS_ROUTER_BASE__ = ipfsPathPrefix + '/'
```

Comment:
nice! this var is the secret ingredient, where did you find it?

Reply:
vuepress bootstrap code.
but `__webpack_public_path__ = ipfsPathPrefix + '/'` this one is even more important without it the router would even load

Reply:
https://github.com/vuejs/vuepress/blob/master/packages/%40vuepress/core/lib/client/app.js#L63

Reply:
I knew there was a `__webpack_public_path__` lurking about but I missed this one! Great work detective 🕵️‍♂️

Reply:
yeah @satazor put me in the right direction with that one i just did the last mile

Reply:
the highest of fives to @satazor! It also makes me wonder if we could make a more generic IPFS plugin for webpack compiled apps using some of these framework config vars. we'd need to do detection for router logic but it doesn't seem impossible now... 🤔

Reply:
Thanks! I also thought about that in the past @cwaring and it’s doable. The route logic detection could also be pluggable, meaning we could have addons/plugins typical frameworks such as Gatsby and Vue press.

I guess it would take some investment time but it would pay off later.

Reply:
nice, if we are delaying the adoption of origin isolation for a while then this might be a worthy endeavour to the community. I see this question crop up all the time.